cardinal/benchmark/benchmark_test.go

```diff
@@ -36,7 +35,7 @@ func setupWorld(t testing.TB, numOfEntities int, enableHealthSystem bool) *testu
 		err := cardinal.RegisterSystems(
 			world,
 			func(wCtx engine.Context) error {
-				q := cardinal.NewSearch(wCtx, filter.Contains(Health{}))
+				q := cardinal.NewSearch(wCtx).Contains(cardinal.SearchComponent[Health]())
```

Comment:
let me see what I can do [](edited-with-Axolo)

Reply:
I did this, but here's a thing that's a bit weird about it. It makes it so that this feature can only be used by using things from two different packages. I don't mind too much though. [](edited-with-Axolo)